src/options.ts

```diff
@@ -34,7 +34,7 @@ export type Target = {
   /**
    * rename
    */
-  rename?: string
+  rename?: string | (() => string)
```

Comment:
I don't think this type is correct.

Reply:
you're right. It needs some params in the function declaration.